ansible/roles/pico-shell/tasks/dependencies.yml

```diff
@@ -58,6 +58,13 @@
 - name: Install requests
   pip:
     name: requests
+    version: 2.19.1
+
+# Needed for pam module, no virtualenv
+- name: Update cryptography
+  pip:
+    name: cryptography
+    version: 1.4
```

Comment:
Without a specific pin, it pulled latest 2.2.2 that did cause an SSL error upon attempted SSH login (as pam_auth.py is called).

Based on the warning message
`/usr/local/lib/python2.7/dist-packages/requests/__init__.py:83: RequestsDependencyWarning: Old version of cryptography ([1, 2, 3]) may cause slowdown.`, 1.4 is the first higher version that does not raise the message in requests. 

I agree that updating pyOpenSSL or otherwise to address the first issue, then pulling latest cryptography would be ideal.

Reply:
After more testing, 1.9 seems to be the highest version of cryptography that can be pinned (as minimal effort without updating openSSL)